### PR TITLE
Adding eBay enduser migration to OTEL blog post

### PR DESCRIPTION
**Pull request by vjsamuel**
Adding eBay tech blog entry for how migration to OTEL was done. 
https://tech.ebayinc.com/engineering/why-and-how-ebay-pivoted-to-opentelemetry/

### CODE REVIEW

File: content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
+---
+title: "Why and How eBay Pivoted to OpenTelemetry"
+linkTitle: eBay OpenTelemetry
+date: 2022-12-13
+author: "[Vijay Samuel](https://github.com/vjsamuel) (eBay)"
+canonical_url: https://tech.ebayinc.com/engineering/why-and-how-ebay-pivoted-to-opentelemetry/
+---
+
+_A version of this article was [originally posted][] on the eBay Tech Blog._
```
Comment (svrnm) [inline content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md:9]: For the last cross-posted blog posts we put this towards the end of the page.

File: content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+
+## Metrics Instrumentation
+
+Metrics instrumentation at eBay has more or less been standardized on Prometheus
+endpoints. Endpoints from various applications are exposed as a result of a
+variety of instrumentation practices such as (but not limited to):
+
+- Official Prometheus clients (including Java, Go, Python and others)
+- [Micrometer](https://micrometer.io/)
+- OTEL SDK with Prometheus exporter
```
Comment (svrnm) [inline content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md:61]:
```suggestion
- OTel SDK with Prometheus exporter
```

**Review by svrnm — CHANGES_REQUESTED** [review]
Minor things, but overall looks good to me.

File: content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md (hunk 1 above)
Comment (vjsamuel) [reply content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md]: done.

File: content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md (hunk 2 above)
Comment (vjsamuel) [reply content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md]: done

**Review by svrnm — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
This is great, thank you so much for posting here as well. We'll get this scheduled for a merge+release

File: content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
+---
+title: "Why and How eBay Pivoted to OpenTelemetry"
+linkTitle: eBay OpenTelemetry
+date: 2022-12-13
```
Comment (svrnm) [inline content/en/blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/index.md:4]:
```suggestion
date: 2022-12-19
```

@vjsamuel please update the date so we can publish it

**Comment by svrnm** [comment]
@open-telemetry/blog-approvers please approve

**Review by austinlparker — APPROVED** [review]

**Pull request merged**